src/lib/theme/_overrides.scss

```diff
@@ -0,0 +1,287 @@
+//
+// All style overrides for dark, light and all components for
+// Vantage UI platform
+// 
+
+$mat-dark-theme-background: map-merge(
+  $mat-dark-theme-background,
+  (
+    card: mat-color($td-slate-dark, 1100),
+    dialog: mat-color($td-slate-dark, 600),
+    backdrop: mat-color($mat-grey, 800),
```

Comment:
What is 'backdrop'? The page background?

Reply:
The overlay over the app when the drawer is open

Reply:
On the design side we've traditionally called this 'overlay'. Will it break things for you guys to change it to that?

Reply:
Literally said "the overlay" haha, lets test it out.